Confus/ConfusServer/Networking/Connection.h

```diff
@@ -60,12 +66,14 @@ namespace ConfusServer
             /// requesting services
             /// </summary>
             void processPackets();
+            std::vector<RakNet::SystemAddress> getOpenConnections();
             /// <summary> Adds a function to the event in the callback function map. </summary>
             /// <param name="a_Event">The server event that should trigger the function.</param>
             /// <param name="a_Function">The function that should be added to the map.</param>
             void addFunctionToMap(unsigned char a_Event, std::function<void(RakNet::Packet* a_Data)> a_Function);
-            void broadcastPacket(RakNet::BitStream* a_Stream, RakNet::AddressOrGUID* a_Excluded = nullptr);
+            void sendMessage(RakNet::BitStream & a_InputStream, PacketReliability a_Reliability);
```

Comment:
You might want to stay consistent and call this sendpacket as well, just as a different overload (or change the name of the other one), though you should change either type of bitstream from pointer to reference or vice versa. I'm not sure why you don't allow for specifying an address, but an even simpler case would be to have the sendPacket method take an optional parameter of the PacketReliability